drivers/Makefile.dep

```diff
@@ -1,875 +1,129 @@
-  FEATURES_REQUIRED += periph_spi
-
-  USEMODULE += xtimer
-  USEMODULE += luid
-  USEMODULE += netif
-  USEMODULE += ieee802154
-  USEMODULE += netdev_ieee802154
 endif
 
 ifneq (,$(filter at86rf%, $(filter-out at86rf215%, $(USEMODULE))))
   USEMODULE += at86rf2xx
-  DEFAULT_MODULE += auto_init_at86rf2xx
-  DEFAULT_MODULE += netdev_ieee802154_oqpsk
-
-  USEMODULE += xtimer
-  USEMODULE += luid
-  USEMODULE += netif
-  USEMODULE += ieee802154
-  USEMODULE += netdev_ieee802154
-
-  # only needed for SPI based variants
-  ifeq (,$(filter at86rfa1 at86rfr2,$(USEMODULE)))
-    FEATURES_REQUIRED += periph_gpio
-    FEATURES_REQUIRED += periph_gpio_irq
-    FEATURES_REQUIRED += periph_spi
-  endif
-endif
-
-ifneq (,$(filter ata8520e,$(USEMODULE)))
-  USEMODULE += xtimer
-  USEMODULE += fmt
-  FEATURES_REQUIRED += periph_gpio
-  FEATURES_REQUIRED += periph_gpio_irq
-  FEATURES_REQUIRED += periph_spi
-endif
-
-ifneq (,$(filter atwinc15x0,$(USEMODULE)))
-  USEMODULE += luid
-  USEMODULE += netdev_eth
-  USEMODULE += xtimer
-  USEPKG += driver_atwinc15x0
-  FEATURES_REQUIRED += periph_gpio
-  FEATURES_REQUIRED += periph_gpio_irq
-  FEATURES_REQUIRED += periph_spi
-endif
-
-ifneq (,$(filter bh1750fvi,$(USEMODULE)))
-  USEMODULE += xtimer
-  FEATURES_REQUIRED += periph_i2c
-endif
-
-ifneq (,$(filter bh1900nux,$(USEMODULE)))
-  FEATURES_REQUIRED += periph_i2c
 endif
 
 ifneq (,$(filter bme680_%,$(USEMODULE)))
-  USEPKG += driver_bme680
   USEMODULE += bme680
-  ifneq (,$(filter saul%,$(USEMODULE)))
-    USEMODULE += xtimer
-  endif
-endif
-
-ifneq (,$(filter bme680_i2c,$(USEMODULE)))
-  FEATURES_REQUIRED += periph_i2c
-endif
-
-ifneq (,$(filter bme680_spi,$(USEMODULE)))
-  FEATURES_REQUIRED += periph_gpio
-  FEATURES_REQUIRED += periph_spi
-endif
-
-ifneq (,$(filter bmp180,$(USEMODULE)))
-  FEATURES_REQUIRED += periph_i2c
-  USEMODULE += xtimer
 endif
 
 ifneq (,$(filter bm%280_spi,$(USEMODULE)))
-  FEATURES_REQUIRED += periph_spi
-  FEATURES_REQUIRED += periph_gpio
   USEMODULE += bmx280
 endif
 
 ifneq (,$(filter bm%280_i2c,$(USEMODULE)))
-  FEATURES_REQUIRED += periph_i2c
   USEMODULE += bmx280
 endif
 
-ifneq (,$(filter bmx280,$(USEMODULE)))
-  USEMODULE += xtimer
-endif
-
-ifneq (,$(filter bmx055,$(USEMODULE)))
-  FEATURES_REQUIRED += periph_i2c
-endif
-
 ifneq (,$(filter cc110%,$(USEMODULE)))
   USEMODULE += cc110x
-  USEMODULE += cc1xxx_common
-  USEMODULE += luid
-  USEMODULE += netif
-  USEMODULE += xtimer
-  FEATURES_REQUIRED += periph_gpio
-  FEATURES_REQUIRED += periph_gpio_irq
-  FEATURES_REQUIRED += periph_spi
-  ifneq (,$(filter gnrc_ipv6,$(USEMODULE)))
-    USEMODULE += gnrc_sixlowpan
-  endif
-endif
-
-ifneq (,$(filter cc2420,$(USEMODULE)))
-  USEMODULE += xtimer
-  USEMODULE += luid
-  USEMODULE += netif
-  USEMODULE += ieee802154
-  USEMODULE += netdev_ieee802154
-  FEATURES_REQUIRED += periph_gpio
-  FEATURES_REQUIRED += periph_gpio_irq
-  FEATURES_REQUIRED += periph_spi
 endif
 
 ifneq (,$(filter ccs811_full,$(USEMODULE)))
```

Comment:
I wonder if it would be better to always address driver variants defined by pseudomodules with `%`, for example.
```suggestion
ifneq (,$(filter ccs811%,$(USEMODULE)))
```
For most of the pseudomodules it's already done in that way, but for some of them direct names are used.

Reply:
Sure, I started with the brute force way but we can refine for sure !

Reply:
Since we touch it once, we should change it with this PR. If I'm right, there are less than 10 occurrences.

Reply:
Addressed, I prefer to use `ccs811_%`.